web/src/navigator.rs

```diff
@@ -95,91 +83,71 @@ impl NavigatorBackend for WebNavigatorBackend {
         target: String,
         vars_method: Option<(NavigationMethod, IndexMap<String, String>)>,
     ) {
-        // If the URL is empty, we ignore the request
-        if url.is_empty() {
-            return;
-        }
```

Comment:
#1102 is currently regressing with this PR, this might be because of the removal of these lines.

Reply:
Good catch, I restored it.